schemas/maas-backend/bookings/bookings-agency-options/request.json

```diff
@@ -59,6 +59,10 @@
         },
         "toRadius": {
           "$ref": "http://maasglobal.com/core/components/units-geo.json#/definitions/distance"
+        },
+        "extensionOptionsForBookingId": {
```

Comment:
i think it would be better in `optionalParameters` group

Reply:
otherwise better name is `bookingIdToExtend`. `extensionOptionsForBookingId` sounds like `extensionOptions`